src/wordList.ts

```diff
@@ -0,0 +1,51 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import { fileURLToPath } from 'url'
+
+const DEFAULTFILEPATH = path.resolve(path.dirname(fileURLToPath(import.meta.url)), '../engmix.txt')
+
+// const DEFAULTFILEPATH = '../engmix.txt' // potentially will change
+
```

Comment:
are we using this ever? @haydenlauritzen

Reply:
fucking yes?

Reply:
What a loser